desktop/apps/auction2/components/range_slider/rc_slider.styl

```diff
@@ -0,0 +1,104 @@
+.rc-slider
```

Comment:
@craigspaeth and I looked at this a little today, but looks like stylus has no ability to import css files inline right now (see https://github.com/stylus/stylus/issues/2155). I pared down the styles from the node module to fit our needs here.